package.json

```
@@ -19,9 +19,8 @@
     "@keep-network/keep-core": "development",
     "@keep-network/keep-ecdsa": "development",
     "@keep-network/random-beacon": "development",
-    "@keep-network/tbtc": "development",
     "@keep-network/tbtc-v2": "development",
-    "@keep-network/tbtc-v2.ts": "2.0.0-dev.0",
+    "@keep-network/tbtc-v2.ts": "2.3.0",
```

Let's allow for minor versions upgrades:
```suggestion
    "@keep-network/tbtc-v2.ts": "^2.3.0",
```

---

Ref commit: 29853e3c